docs/book/src/tasks/experimental-features/cluster-class/write-clusterclass.md

```diff
@@ -438,6 +438,81 @@ spec:
         template: "{{ .cluster.name }}-{{ .machinePool.topologyName }}-{{ .random }}"
 ```
 
+### Defining a custom namespace for ClusterClass object
+
+When referencing a `ClusterClass` from a `Cluster`, user may want to reuse existing template from a `Cluster` in a different namespace. To uniquely identify the `ClusterClass`, a `NamespaceName` ref is constructed from combination of:
```

Comment:
```suggestion
As a user, I may need to create a `Cluster` from a `ClusterClass` object that exists only in a different namespace. To uniquely identify the `ClusterClass`, a `NamespaceName` ref is constructed from combination of:
```
This is just a suggestion but I think this could be reworded to make it clearer. What do you think?